src/service/file-service.js

```diff
@@ -1,3 +1,4 @@
+import { glob } from 'glob';
```

Comment:
Can you please give more explanation why did you add this. I would like to avoid adding new dependencies in project unless it's really necessary

Reply:
It's better to have names of the files in the pending storage in the format: `blockchain:contract:tokenId:stateId`, because then we can easily check if given *state (hash)* is pending or not without reading the file. Glob is needed to keep the possibility to get pending state just by using blockchain/contract/tokenId, because we don't expect 1 Knowledge Asset having more than 1 pending state (it's not possible)

Reply:
Not sure that i understand, how can you check if given state is pending or not without reading the file. Previously we were checking if file exists on certain path, and reading it just by passing that path. Why is glob different?